test/cluster/lwt/test_lwt_during_tablets_migration.py

```diff
@@ -0,0 +1,303 @@
+        with open(filename, "w", encoding="utf-8") as f:
+            json.dump(self.history, f, indent=2)
+        logger.info(f"Saved {len(self.history)} history entries to {filename}")
+
+
+async def get_non_replica_server(manager: ManagerClient, servers, replicas):
+    replica_host_ids = {replica[0] for replica in replicas}
+    for server in servers:
+        host_id = await manager.get_host_id(server.server_id)
+        if host_id not in replica_host_ids:
+            return server
+    raise ValueError("No non-replica servers found")
+
+
+async def migrate_tablet(manager: ManagerClient, src_server, dst_server, ks: str, tbl: str, token: int, replicas):
+    logger.info(f"Starting tablet migration to {dst_server.ip_addr}")
+
+    dst_host_id = await manager.get_host_id(dst_server.server_id)
+    src_replica = replicas[0]
+
+    await manager.api.move_tablet(
+        src_server.ip_addr, ks, tbl,
+        src_replica[0], src_replica[1],
+        dst_host_id, 0, token
+    )
+
+    current_replicas = await get_tablet_replicas(manager, src_server, ks, tbl, token)
+    assert any(repl[0] == dst_host_id for repl in current_replicas), "Tablet migration failed"
+    logger.info(f"Tablet migration completed to {dst_server.ip_addr}")
+
+
+async def get_token_for_pk(cql, ks: str, tbl: str, pk: int) -> int:
+    stmt = SimpleStatement(
+        f"SELECT token(pk) AS tk FROM {ks}.{tbl} WHERE pk = %s",
+        consistency_level=ConsistencyLevel.ONE,
+    )
+    row = (await cql.run_async(stmt, [pk]))[0]
+    return row.tk
+
+
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
+@skip_mode('debug', 'debug mode is too slow for this test')
+async def test_multi_column_lwt_during_migration(manager: ManagerClient):
+    """Test multi-column LWT pattern during tablet migration"""
+
+    # Setup cluster
+    cfg = {
+        "enable_user_defined_functions": False,
+        "tablets_mode_for_new_keyspaces": "enabled"
+    }
+    property_files = [{"dc": "dc1", "rack": f"rack{(i % 2) + 1}"} for i in range(6)]
+    servers = await manager.servers_add(6, config=cfg, property_file=property_files)
+
+    for server in servers:
+        await manager.api.disable_tablet_balancing(server.ip_addr)
+
+    async with new_test_keyspace(
+        manager,
+        "WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 2} AND tablets = {'initial': 5}"
+    ) as ks:
+
+        tester = MultiColumnLWTTester(manager, ks, "lwt_table")
+        await tester.create_schema()
+        await tester.initialize_rows()
+        await tester.prepare_statements()
+        await tester.start_workers()
+
+        try:
+            # Phase 1: Pre-migration workload
+            await asyncio.sleep(WORKLOAD_SEC / 3)
+            hot_pk = max(tester.success_counts, key=lambda k: sum(tester.success_counts[k]))
+            token = await get_token_for_pk(tester.cql, ks, tester.tbl, hot_pk)
```

Comment:
I think we should just run migrations continuously in a loop for WORKLOAD_SEC instead of just sleeping. Peek a random tablet/replica, migrate it to some other replica, repeat. Ideally, we should be doing split/merges as well, but probably in a follow-up.